mets.md

```diff
@@ -141,41 +210,7 @@ encodings of the same page.
 </mets:structMap>
 ```
 
-## Images and coordinates
-
-Coordinates are always absolute, i.e. relative to extent defined in the `imageWidth`/`imageHeight` attribute of the nearest `<pc:Page>`.
-
-When a processor wants to access the image of a layout element like a TextRegion or TextLine, the algorithm should be:
-
-- If the element in question has an attribute `imageFilename`, resolve this value
-- If the element has a `<pc:Coords>` subelement, resolve by passing the attribute `imageFilename` of the nearest `<pc:Page>` and the `points` attribute of the `<pc:Coords>` element
-
-## One page in one PAGE
-
-A single PAGE XML file represents one page in the original document.
-
-Every `<pc:Page>` element MUST have an attribute `image` which MUST always be the source image.
-
-The PAGE XML root element `<pc:PcGts>` MUST have exactly one `<pc:Page>`.
-
-## Media Type for PAGE XML
-
-Every `<mets:file>` representing a PAGE document MUST have its `MIMETYPE` attribute set to `application/vnd.prima.page+xml`.
-
-## Always use URL everywhere
-
-Always use URL. If it's a local file, prefix absolute path with `file://`.
-
-### Example
-
-```xml
-<mets:fileGrp USE="OCR-D-SEG-BLOCK">
-    <mets:file ID="OCR-D-SEG-BLOCK_0001" MIMETYPE="application/vnd.prima.page+xml">
-        <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="file:///path/to/workingDir/segmentation/block/page_0001.xml" />
-    </mets:file>
-</mets:fileGrp>
-```
-
+#PAGE and METS
```

Comment:
Maybe we can reverse the shifting of sections to focus on the changes by this PR?

Reply:
You're right.